docs/en/cost.md

```diff
@@ -0,0 +1,18 @@
+You are responsible for the cost of Amazon cloud technology services used when running this solution. As of November 2021, with the default settings in the us-west-2 region, for 540p to 4K super resolution tasks, the estimated cost of running the solution to process the original video is approximately $8.86 per hour.
+
+## Cost estimate
```

Comment:
suggest more specific when comes to cost, like cost estimation of AWS Batch, since most of the time we don't need or can't iterate all possible service involved, we just need those contribute most of the cost like AWS Batch in such case